### PR TITLE
[Edgecore][as9926 24db] Add BMC detection in __init__.py

### PR DESCRIPTION
**Pull request by acctalex**

### CODE REVIEW

File: packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py [hunk 1]
```diff
@@ -1,13 +1,80 @@
...
+from itertools import chain
 from onl.platform.base import *
 from onl.platform.accton import *
 
+init_ipmi_dev = [
+    'echo "remove,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod',
+    'echo "add,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod']
+
+ATTEMPTS = 5
+INTERVAL = 3
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py:12]: Why these numbers? Please add a comment why it does not work at first try, and why these values were chosen.

File: packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py [hunk 2]
```diff
@@ -1,13 +1,80 @@
+import commands
+import time
+from itertools import chain
 from onl.platform.base import *
 from onl.platform.accton import *
 
+init_ipmi_dev = [
+    'echo "remove,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod',
+    'echo "add,kcs,i/o,0xca2" > /sys/module/ipmi_si/parameters/hotmod']
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py:9]: Should the ] be on the next line?

File: packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py [hunk 3]
```diff
@@ -1,13 +1,80 @@
...
+        time.sleep(interval)
+
+    return (False, ATTEMPTS * interval)
+
+def init_ipmi_oem_cmd():
+    attempts = ATTEMPTS
+    interval = INTERVAL
+
+    while attempts:
+        status, output = commands.getstatusoutput('ipmitool raw 0x34 0x95')
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py:35]: Please add a comment, what these commands do.

File: packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py [hunk 4]
```diff
@@ -38,5 +105,4 @@ def baseconfig(self):
             subprocess.call('echo port%d > /sys/bus/i2c/devices/%d-0050/port_name' % (port, port+8), shell=True)
 
 
-        return True
-
+        return True
```
Comment (paulmenzel) [inline packages/platforms/accton/x86-64/as9926-24db/platform-config/r0/src/python/x86_64_accton_as9926_24db_r0/__init__.py]: Please squash the next commit into this one.

**Review by paulmenzel — CHANGES_REQUESTED** [review]
Thank you for your patch.

1.  Please document the BMC model in the commit message.
1.  Please fix the typo *interface* in the commit message.
1.  Please describe the problem in the commit message, and why loading the BMC module once is not enough. Why can loading it fail?

**Review by sonoble — APPROVED** [review]
LGTM

**Pull request merged**